src/io/import_off.cc

```diff
@@ -11,6 +11,36 @@
 // References:
 // http://www.geomview.org/docs/html/OFF.html
 
+#if !defined(BOOST_ENDIAN_BIG_BYTE_AVAILABLE) && !defined(BOOST_ENDIAN_LITTLE_BYTE_AVAILABLE)
+#error Byte order undefined or unknown. Currently only BOOST_ENDIAN_BIG_BYTE and BOOST_ENDIAN_LITTLE_BYTE are supported.
+#endif
+
+// Specs says data is big-endian, however PMP-library used little-endian
```

Comment:
I would say this is a PMP bug. Should we really follow that example and support software outputting buggy files?

Reply:
It depends if it's been used widely or not…